code/modules/integrated_electronics/core/printer.dm

```diff
@@ -273,11 +272,11 @@
 					if(!materials.use_amount_type(program["metal_cost"], MAT_METAL))
 						to_chat(usr, "<span class='warning'>You need [program["metal_cost"]] metal to build that!</span>")
 						return
-					var/cloning_time = program["metal_cost"] / 150
+					var/cloning_time = round(program["metal_cost"] / 15)
 					cloning_time = min(cloning_time, MAX_CIRCUIT_CLONE_TIME)
 					cloning = TRUE
-					clone_countdown = cloning_time
-					to_chat(usr, "<span class='notice'>You begin printing a custom assembly. This will take approximately [round(cloning_time / 60, 0.1)] minute(s). You can still print \
+					clone_countdown = world.time + cloning_time
+					to_chat(usr, "<span class='notice'>You begin printing a custom assembly. This will take approximately [DisplayTimeText(cloning_time)]. You can still print \
```

Comment:
should this also be `cloning_time + world.time`?

Reply:
err, minus not plus

Reply:
It didn't seem to be the case in testing. I'm not in a spot to test it right now though, so it's your call.

Reply:
on another glance, I misread. confused the two cloning vars.